app/src/features/ai/components/ai-options-settings-card.tsx

```
@@ -0,0 +1,467 @@
+                        aiUserPreferences={aiUserPreferences}
+                        editServerIndex={editServerIndex === undefined ? undefined : editServerIndex}
+                        onOpenChange={(open) => {
+                            setDialogOpen(open);
+
+                            if (!open)
+                                setEditServerIndex(undefined);
+                        }}
+                        onSubmit={async (newServer) => {
+                            let updatedServers;
+
+                            updatedServers
```

ESLint

[error] 'updatedServers' is never reassigned. Use 'const' instead.

(prefer-const)